src/find_deletions/deletion_finding_and_printing.cpp

```diff
@@ -1,7 +1,9 @@
-#include <fstream>
-
 #include "find_deletions/deletion_finding_and_printing.hpp"
-#include "variant_parser/variant_record.hpp"
+
+#include <iostream> // for std::cout
```

Comment:
usually std includes are located above local includes

Reply:
I thougt its better for the understandig to put the include of the associated hpp file above everything.

Reply:
Ah sure, that's also a valid style :)